crates/gpui/src/platform/windows/window.rs

```diff
@@ -1221,10 +1221,10 @@ fn set_window_composition_attribute(hwnd: HWND, color: Option<Color>, state: u32
     unsafe {
         type SetWindowCompositionAttributeType =
             unsafe extern "system" fn(HWND, *mut WINDOWCOMPOSITIONATTRIBDATA) -> BOOL;
-        let module_name = PCSTR::from_raw("user32.dll\0".as_ptr());
+        let module_name = PCSTR::from_raw(c"user32.dll".as_ptr());
```

Comment:
```suggestion
        let module_name = windows::core::w!("user32.dll");
```
Try using this macro.

Reply:
Never mind, it's `PCSTR` not `PCWSTR`